catalogue/webapp/services/catalogue/works.ts

```diff
@@ -64,6 +64,30 @@ const redirect = (id: string, status = 302): CatalogueApiRedirect => ({
   status,
 });
 
+/** Creates the YYYY-MM-DD date string we pass to the API.
+ *
+ * Note: the filter GUI expects users to enter dates as a four-digit year (e.g. 1939).
+ * We pin to the start/end of the year so that the range is inclusive.
+ *
+ * e.g. a user who searches for works 'to 2001' should find works created in 2001.
+ */
+function toIsoDateString(
+  s: string | undefined,
+  range: 'to' | 'from'
+): string | undefined {
+  if (s) {
+    try {
+      const d = new Date(s);
+      return range === 'from'
+        ? `${d.getUTCFullYear()}-01-01`
+        : `${d.getUTCFullYear()}-12-31`;
+    } catch (e) {
```

Comment:
I learned recently that if you're not going to do anything with the `catch` argument, you don't need to include it in a JS try/catch block

Reply:
Ooh, good to know, thanks.